src/lesson/lesson01.js

```diff
@@ -461,6 +625,69 @@ function Lesson01() {
   this.addStep(
     new Lesson.LessonStep(
       <p>
+        Salve os dois robôs!
+      </p>,
+      <p>
+        Alguns de nossos robôs recebem comandos do mesmo computador e, por isso,
+         o programa que for escrito para salvá-los deve ser único e deve funcionar
+         igualmente para os dois robôs. Veja neste exemplo: o programa WLWW
+         salva o primeiro robô, mas faz com que o segundo seja atingido por um meteoro.
+         Já o programa LLWW salva o segundo robô, mas faz com que o primeiro
+         seja atingido por um meteoro. O único programa que salva os dois
+         robôs, ao mesmo tempo, é WRLW (ou WRL).
+      </p>,
+      new Lesson01ExerciseStepPlayerMulti(true, 2, [5, 5], [5, 5], [2, 2],
+                                     [[6, 8, 12, 15, 18], [8, 11, 19]]),
```

Comment:
The indentation is non-standard here: the `[` should align with the `t` in `true` (same for the other added steps).